components/eam/src/chemistry/modal_aero/modal_aero_initialize_data.F90

```diff
@@ -609,13 +609,14 @@ subroutine modal_aero_initialize(pbuf2d, imozart, species_class)
           call modal_aero_newnuc_init( mam_amicphys_optaa )
           call modal_aero_amicphys_init( imozart, species_class,n_so4_monolayers_pcage_in )
        else
-          call modal_aero_rename_init
-          !   calcsize call must follow rename call
-          call modal_aero_calcsize_init( pbuf2d, species_class )
-          call modal_aero_gasaerexch_init
-          !   coag call must follow gasaerexch call
-          call modal_aero_coag_init
-          call modal_aero_newnuc_init( mam_amicphys_optaa )
+        ! call modal_aero_rename_init
```

Comment:
We don't need this "if" condition anymore.

Reply:
Removed.